components/organisms/BookPreview.tsx

```diff
@@ -110,6 +137,15 @@ export default function BookPreview({
               onClick={handle(onBookEditClick)}
             />
           )}
+          {onBookLinkClick && (
```

Comment:
[components/templates/Books.tsx R133](https://github.com/npocccties/chibichilo/pull/434/files#diff-8718b4609de544ee921d3e49bdceeafc95c4403b6f4a7f700f2408dece5011efR133)
で親から条件演算子で制御を行っていますがいまのところ下記のようにこのコンポーネント内の振る舞いとして閉じていても良さそうに見えましたがいかがでしょうか。
「提供中」かつ「このブックを提供」するユースケースは無さそうに考えますがいかがでしょうか。

```suggestion
          {!linked && onBookLinkClick && (
```

```diff
-                onBookLinkClick={!linked ? onBookLinkClick : undefined}
+                onBookLinkClick={onBookLinkClick}
```

Reply:
> 「提供中」かつ「このブックを提供」するユースケースは無さそう

ないです。おっしゃるとおり親から制御する必要はないです。そのように変更しますね